docs/source/optimizations/ibm/ibmq_compile_qss.ipynb

```diff
@@ -21,9 +21,37 @@
     "</a>"
```

Comment:
<p>Maybe worth reducing the output here as well to match the css notebook version:</p><pre class="ql-syntax" spellcheck="false">%pip install -q qiskit-superstaq[examples]
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/539/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done.